server/lib/routes/get-metrics-errors.js

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+var logger = require('mozlog')('server.metrics-errors');
+var request = require('request');
+
+var config = require('../configuration');
+var sentryConfig = config.get('sentry');
+
+var API_KEY = sentryConfig.api_key;
+
+/**
+ * Reports errors to Sentry
+ * @param {String} url
+ *          Url from the error request.
+ */
+function reportError(url) {
+  var indexOfQuery = url.indexOf('?');
+
+  if (sentryConfig && API_KEY && indexOfQuery >= 0) {
+    // send the request to sentry
+    var query = url.substr(indexOfQuery + 1);
+    // set API_KEY using the server
+    query = query.replace('__API_KEY__', API_KEY);
+
+    process.nextTick(function () {
```

Comment:
YES! We should do this with the other "proxy" endpoints too. Free up the main thread, send the request when it gets around to it.